README.md

```diff
@@ -19,6 +20,7 @@
 
 ## Current News
 * November 14, 2023 - [Announcing .NET MAUI in .NET 8](https://devblogs.microsoft.com/dotnet/announcing-dotnet-maui-in-dotnet-8/)
+* November 8, 2024 - [Announcing .NET MAUI in .NET 9](https://learn.microsoft.com/en-us/dotnet/maui/whats-new/dotnet-9?view=net-maui-9.0)
```

Comment:
```suggestion
* November 13, 2024 - [What's new in .NET MAUI for .NET 9](https://learn.microsoft.com/dotnet/maui/whats-new/dotnet-9)
* November 12, 2024 - [Announcing .NET 9](https://devblogs.microsoft.com/dotnet/announcing-dotnet-9/#.net-maui-–-enhancing-multi-platform-app-development)
```

Let's just remove the .NET 8 link and besides the "whats changed" link also add the announcing .NET 9 blog post